### PR TITLE
fix(ui): Various minor UI fixes

### LINKED ISSUE #5324: Scroll up button in logs viewer
If you are trying to resolve an environment-specific issue or have a one-off question about the edge case that does not require a feature then please consider asking a question in argocd slack [channel](https://argoproj.github.io/community/join-slack).

Checklist:

* [x] I've searched in the docs and FAQ for my answer: https://bit.ly/argocd-faq.
* [x] I've included steps to reproduce the bug.
* [x] I've pasted the output of `argocd version`.

**Describe the bug**

#5233 introduced a scroll down button, but once one clicks it, there's no "scroll up" button

**To Reproduce**

1. Visit https://cd.apps.argoproj.io/applications/argo-cd?node=%2FPod%2Fargocd%2Fargocd-server-cb9988685-8d7rt%2F0&tab=logs (your node may vary)
2. Click `v`

**Expected behavior**

Have a `^` button

**Screenshots**

image

**Version**

```shell
{
    "Version": "v1.9.0+7165b45",
    "BuildDate": "2021-01-26T21:23:37Z",
    "GitCommit": "7165b45aeaa15044cec514e7de3d300d448b2df9",
    "GitTreeState": "clean",
    "GoVersion": "go1.14.12",
    "Compiler": "gc",
    "Platform": "linux/amd64",
    "KsonnetVersion": "v0.13.1",
    "KustomizeVersion": "v3.8.1 2020-07-16T00:58:46Z",
    "HelmVersion": "v3.4.1+gc4e7485",
    "JsonnetVersion": "v0.17.0"
}
```

**Logs**

```
Paste any relevant application logs here.
```

### LINKED ISSUE #5325: k8s client version is blank after removing client binary
If you are trying to resolve an environment-specific issue or have a one-off question about the edge case that does not require a feature then please consider asking a question in argocd slack [channel](https://argoproj.github.io/community/join-slack).

Checklist:

* [x] I've searched in the docs and FAQ for my answer: https://bit.ly/argocd-faq.
* [x] I've included steps to reproduce the bug.
* [x] I've pasted the output of `argocd version`.

**Describe the bug**

#5101 removed `kubectl` from the image leaving a blank version box.

**To Reproduce**

1. Load https://cd.apps.argoproj.io/applications/argo-cd
2. Click the version (v1.9.0+7165b45)

**Expected behavior**

Report k8s client version

**Screenshots**

image

**Version**

```shell
{
    "Version": "v1.9.0+7165b45",
    "BuildDate": "2021-01-26T21:23:37Z",
    "GitCommit": "7165b45aeaa15044cec514e7de3d300d448b2df9",
    "GitTreeState": "clean",
    "GoVersion": "go1.14.12",
    "Compiler": "gc",
    "Platform": "linux/amd64",
    "KsonnetVersion": "v0.13.1",
    "KustomizeVersion": "v3.8.1 2020-07-16T00:58:46Z",
    "HelmVersion": "v3.4.1+gc4e7485",
    "JsonnetVersion": "v0.17.0"
}
```

**Logs**

```
Paste any relevant application logs here.
```

### LINKED ISSUE #5327: buttons should not change size when clicked nor after a timeout
If you are trying to resolve an environment-specific issue or have a one-off question about the edge case that does not require a feature then please consider asking a question in argocd slack [channel](https://argoproj.github.io/community/join-slack).

Checklist:

* [x] I've searched in the docs and FAQ for my answer: https://bit.ly/argocd-faq.
* [x] I've included steps to reproduce the bug.
* [x] I've pasted the output of `argocd version`.

**Describe the bug**

Copy/Copied button changes width

**To Reproduce**

1. Visit https://cd.apps.argoproj.io/applications/argo-cd?node=%2FPod%2Fargocd%2Fargocd-redis-ha-server-2%2F0&tab=logs (a log viewer)
2. Position mouse near edge of things and click
3. position and click

**Expected behavior**

Buttons shouldn't change size/position randomly -- it's dangerous/bad practice.

**Screenshots**

https://user-images.githubusercontent.com/2119212/105938907-24f51f80-6026-11eb-9b01-145ac8806ea3.mov

**Version**

```shell
{
    "Version": "v1.9.0+7165b45",
    "BuildDate": "2021-01-26T21:23:37Z",
    "GitCommit": "7165b45aeaa15044cec514e7de3d300d448b2df9",
    "GitTreeState": "clean",
    "GoVersion": "go1.14.12",
    "Compiler": "gc",
    "Platform": "linux/amd64",
    "KsonnetVersion": "v0.13.1",
    "KustomizeVersion": "v3.8.1 2020-07-16T00:58:46Z",
    "HelmVersion": "v3.4.1+gc4e7485",
    "JsonnetVersion": "v0.17.0"
}
```

**Logs**

```
Paste any relevant application logs here.
```

### LINKED ISSUE #5328: Hand indicator should indicate clicking does something
If you are trying to resolve an environment-specific issue or have a one-off question about the edge case that does not require a feature then please consider asking a question in argocd slack [channel](https://argoproj.github.io/community/join-slack).

Checklist:

* [x] I've searched in the docs and FAQ for my answer: https://bit.ly/argocd-faq.
* [x] I've included steps to reproduce the bug.
* [x] I've pasted the output of `argocd version`.

**Describe the bug**

Sometimes nothing happens when one has a hand cursor and clicks

**To Reproduce**

1. Visit https://cd.apps.argoproj.io/applications/argocd-image-updater?view=pods
2. Hover over various elements
3. When one has a hand 👆  cursor, click things
4. Change to the other views
5. Repeat

**Expected behavior**

👆 Should indicate a clickable object

**Screenshots**

https://user-images.githubusercontent.com/2119212/105939428-2ecb5280-6027-11eb-87e8-5a1b07e649e0.mov

**Version**

```shell
{
    "Version": "v1.9.0+7165b45",
    "BuildDate": "2021-01-26T21:23:37Z",
    "GitCommit": "7165b45aeaa15044cec514e7de3d300d448b2df9",
    "GitTreeState": "clean",
    "GoVersion": "go1.14.12",
    "Compiler": "gc",
    "Platform": "linux/amd64",
    "KsonnetVersion": "v0.13.1",
    "KustomizeVersion": "v3.8.1 2020-07-16T00:58:46Z",
    "HelmVersion": "v3.4.1+gc4e7485",
    "JsonnetVersion": "v0.17.0"
}
```

**Logs**

```
Paste any relevant application logs here.
```

### LINKED ISSUE #5329: Change `Compiler` to `Go Compiler` in About
If you are trying to resolve an environment-specific issue or have a one-off question about the edge case that does not require a feature then please consider asking a question in argocd slack [channel](https://argoproj.github.io/community/join-slack).

Checklist:

* [x] I've searched in the docs and FAQ for my answer: https://bit.ly/argocd-faq.
* [x] I've included steps to reproduce the bug.
* [x] I've pasted the output of `argocd version`.

**Describe the bug**

`Compiler` is a bit vague for end users, adding the word `Go` before it would improve user comprehension at minimal cost

**To Reproduce**

1. Visit https://cd.apps.argoproj.io/help
2. Click the version under the logo
3. Wonder what `Compiler` `gc` is

**Expected behavior**

See `Go Compiler` `gc`

**Screenshots**

image

**Version**

```shell
{
    "Version": "v1.9.0+7165b45",
    "BuildDate": "2021-01-26T21:23:37Z",
    "GitCommit": "7165b45aeaa15044cec514e7de3d300d448b2df9",
    "GitTreeState": "clean",
    "GoVersion": "go1.14.12",
    "Compiler": "gc",
    "Platform": "linux/amd64",
    "KsonnetVersion": "v0.13.1",
    "KustomizeVersion": "v3.8.1 2020-07-16T00:58:46Z",
    "HelmVersion": "v3.4.1+gc4e7485",
    "JsonnetVersion": "v0.17.0"
}
```

**Logs**

```
Paste any relevant application logs here.
```

### PR DESCRIPTION
**Pull request by rbreeze**
Fix #5329, fix #5328, fix #5327, fix #5324

Signed-off-by: Remington Breeze <remington@breeze.software>

Checklist:

* [x] Either (a) I've created an [enhancement proposal](https://github.com/argoproj/argo-cd/issues/new/choose) and discussed it with the community, (b) this is a bug fix, or (c) this does not need to be in the release notes.
* [x] The title of the PR states what changed and the related issues number (used for the release note).
* [x] I've updated both the CLI and UI to expose my feature, or I plan to submit a second PR with them.
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [x] Optional. My organization is added to USERS.md.
* [x] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/tree/master/community#contributing-to-argo)
* [ ] My build is green ([troubleshooting builds](https://argoproj.github.io/argo-cd/developer-guide/ci/)).

### CODE REVIEW

**Review by mayzhang2000 — APPROVED** [review]

File: ui/src/app/shared/components/version-info/version-info-panel.tsx [hunk 1]
```diff
@@ -42,14 +42,13 @@ export class VersionPanel extends React.Component<VersionPanelProps, {copyState:
...
-            'Go': version.GoVersion,
-            'Compiler': version.Compiler,
+            'Go Version': version.GoVersion,
+            'Go Compiler': version.Compiler,
             'Platform': version.Platform,
             'ksonnet': version.KsonnetVersion,
             'jsonnet': version.JsonnetVersion,
             'kustomize': version.KustomizeVersion,
-            'Helm': version.HelmVersion,
-            'kubectl': version.KubectlVersion
```
Comment (alexmt) [inline ui/src/app/shared/components/version-info/version-info-panel.tsx:52]: Kubectl version is still required.

**Review by alexmt — CHANGES_REQUESTED** [review]

File: ui/src/app/shared/components/version-info/version-info-panel.tsx (hunk 1 above)
Comment (alexmt) [reply ui/src/app/shared/components/version-info/version-info-panel.tsx:52]: see https://github.com/argoproj/argo-cd/issues/5325#issuecomment-768093326

Comment (rbreeze) [reply ui/src/app/shared/components/version-info/version-info-panel.tsx:52]: will revert back, can go in another PR

**Review by alexmt — APPROVED** [review]
LGTM

**Comment by jsoref** [comment]
@rbreeze: Can you please remove the issue from the [description](https://github.com/argoproj/argo-cd/pull/5337#issue-562733230) that you dropped per https://github.com/argoproj/argo-cd/pull/5337#discussion_r565735065?

Sorry about that. I've updated the description of that issue to match its updated description.

**Review by jsoref — APPROVED** [review]
Seems reasonable to me.

Sadly, while I can more or less read the code, it's much easier for me to work from live previews (that's a feature we really want our ArgoCD to manage for our own software...).

**Pull request merged**